scripts/CEE/get-kafka-instance-state/get-kafka-instance-state.sh

```diff
@@ -23,8 +22,16 @@ function main() {
   local kafkaStatefulSet
   kafkaStatefulSet=$(oc -n "${resource_namespace}" get statefulset -l app.kubernetes.io/name=kafka -o name)
 
-  oc -n "${resource_namespace}" adm inspect ns/"${resource_namespace}" "${managedKafkaCR}" "${kafkaCR}" \
-    --dest-dir "${inspectDir}" > "${inspectDir}"/ocadm.log 2>&1
+  if [[ -v since ]]; then
```

Comment:
https://deviq.com/principles/dont-repeat-yourself applies.
Having to repeat the same thing over and over, changing one thing is a bad smell.   Always look for a better way to solve the problem.

bash supports arrays... and appending to arrays

arrVar=("AC" "TV" "Mobile" "Fridge" "Oven" "Blender")
arrVar+=("Dish Washer")
echo "${arrVar[@]}"

so how about creating an array first with all your common arguments, then use conditional logic append either since or since_time arguments parts.  Finally execute passing the array

```
oc -n "${resource_namespace}" adm inspect "${arrVar[@]}"
```

Reply:
Thank you Keith, that's much more efficient. I've pushed some changes now!

Reply:
change looks good.  pushed one more change to include --dest-dir in inspectOps too. okay to resolve comment.